spec.md

```diff
@@ -147,12 +147,20 @@ that contains both context and data).
   * MUST be a non-empty string
 
 ### source
-* Type: Object
-* Description: This describes the software instance that emits the event at
-  runtime (i.e. the producer). It contains sub-properties (listed below)
+* Type: String
+* Description: This property identifies the source of the event as defined
+  by the publishing organization. The value SHOULD be a URI. The source property
```

Comment:
Who is "publishing organization"? Is this the `namespace` field? If so, we should reuse terms. If not, it sounds like the definition is circular: the publisher (identified by `source`) determines the format and meaning of `source`.

A litmus test I frequently use is an on-prem install of open source software. Which field is apache.org and which field is the domain/IP address where the software is hosted?

Reply:
"Publishing organization" is whoever writes the event. It's the namespace owner and we should add that to the definitions. "namespace" per-se is a qualifier that represents the publishing organization's chosen scope.